lib/new_relic/agent/instrumentation/elasticsearch.rb

```diff
@@ -0,0 +1,31 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative 'elasticsearch/instrumentation'
+require_relative 'elasticsearch/chain'
+require_relative 'elasticsearch/prepend'
+
+DependencyDetection.defer do
+  named :elasticsearch
+
+  depends_on do
+    defined?(::Elasticsearch)
+  end
+
+  executes do
+    ::NewRelic::Agent.logger.info('Installing elasticsearch instrumentation')
+
+    to_instrument = if ::Gem::Version.create(::Elasticsearch::VERSION) < ::Gem::Version.create("8.0.0")
```

Comment:
Do you happen to know if `.create` is an alias for `.new` or if it does anything different? (Asking because I usually use `.new` and was curious when I saw `.create`)

Reply:
I'm not sure, I think I've only seen create being used in the agent. 
[Looking](https://ruby-doc.org/stdlib-2.5.0/libdoc/rubygems/rdoc/Gem/Version.html#method-c-create) at the source code section though it looks like they are slightly different, but create does use new as part of it.